factors_project/factors_all.py

```diff
@@ -1279,3 +1279,66 @@ def compute(self, today, assets, out, tot_assets, tot_liabilities, r, close):
+    'Capex to Assets': Factors.Capex_To_Assets,
+    'Capex to Cashflows': Factors.Capex_To_Cashflows,
+    'Capex to Sales': Factors.Capex_To_Sales,
+    'Cashflows to Assets': Factors.Cashflows_To_Assets,
+    'Current Ratio': Factors.Current_Ratio,
+    'Debt to Asset Ratio': Factors.Debt_To_Asset_Ratio,
+    'Dividend Growth': Factors.Dividend_Growth,
+    'Dividend Payout Ratio': Factors.Dividend_Payout_Ratio,
+    'Dividend Yield': Factors.Dividend_Yield,
+    'Downside Beta': Factors.Downside_Beta,
+    'Downside Risk': Factors.Downside_Risk,
+    'EBITDA Yield': Factors.EBITDA_Yield,
+    'EBIT to Assets': Factors.EBIT_To_Assets,
+    'EPS Growth 12M': Factors.EPS_Growth_12M,
+    'EV to Cashflows': Factors.EV_To_Cashflows,
+    'EV to EBITDA': Factors.EV_To_EBITDA,
+    'Gross Income Margin': Factors.Gross_Income_Margin,
+    'Index Beta': Factors.Index_Beta,
+    'Interest Coverage': Factors.Interest_Coverage,
+    'Log Market Cap': Factors.Log_Market_Cap,
+    'Log Market Cap Cubed': Factors.Log_Market_Cap_Cubed,
+    'MACD Signal Line': Factors.MACD_Signal_10d,
+    'Market Cap': Factors.Market_Cap,
+    'Mean Reversion 1M': Factors.Mean_Reversion_1M,
+    'Mertons Distance to Default': Factors.Mertons_DD,
+    'Moneyflow Volume 5D': Factors.Moneyflow_Volume_5d,
+    'Net Debt Growth 3M': Factors.Net_Debt_Growth_3M,
+    'Net Income Margin': Factors.Net_Income_Margin,
+    'Operating Cashflows to Assets': Factors.Operating_Cashflows_To_Assets,
+    'Percent Above Low': Factors.Percent_Above_Low,
+    'Percent Below High': Factors.Percent_Below_High,
+    'Price Momentum 12M': Factors.Price_Momentum_12M,
+    'Price Momentum 1M': Factors.Price_Momentum_1M,
+    'Price Momentum 3M': Factors.Price_Momentum_3M,
+    'Price Momentum 6M': Factors.Price_Momentum_6M,
+    'Price Oscillator': Factors.Price_Oscillator,
+    'Price to Book': Factors.Price_To_Book,
+    'Price to Diluted Earnings': Factors.Price_To_Diluted_Earnings,
+    'Price to Earnings': Factors.Price_To_Earnings,
+    'Price to Forward Earnings': Factors.Price_To_Forward_Earnings,
+    'Price to Free Cashflows': Factors.Price_To_Free_Cashflows,
+    'Price to Operating Cashflows': Factors.Price_To_Operating_Cashflows,
+    'Price to Sales': Factors.Price_To_Sales,
+    'Retained Earnings to Assets': Factors.Retained_Earnings_To_Assets,
+    'Return on Total Assets': Factors.Return_On_Total_Assets,
+    'Return on Total Equity': Factors.Return_On_Total_Equity,
+    'Return on Invest Capital': Factors.Return_On_Total_Invest_Capital,
+    '39 Week Returns': Factors.Returns_39W,
+    'Sales Growth 12M': Factors.Sales_Growth_12M,
+    'Sales Growth 3M': Factors.Sales_Growth_3M,
+    'Stochastic Oscillator': Factors.Stochastic_Oscillator,
+    'Trendline': Factors.Trendline,
+    'Vol 3M': Factors.Vol_3M,
+    'Working Capital to Assets': Factors.Working_Capital_To_Assets,
+    'Working Capital to Sales': Factors.Working_Capital_To_Sales,
+    
+}
```

Comment:
Can we add a newline at the end? It's nitpicky but probably worth the quick change.